JASP-Desktop/analysis/analysisform.cpp

```diff
@@ -218,11 +218,11 @@ void AnalysisForm::addControl(JASPControlBase *control)
 			if (label.isEmpty())
 				label = control->property("title").toString();
 
-			if (label.isEmpty())	_formErrorMessages.append(tr("Control with label '%1' has no name").arg(label));
-			else					_formErrorMessages.append(tr("A control has no name"));
+			if (label.isEmpty())	_formErrors.append(tr("Control with label '%1' has no name").arg(label));
+			else					_formErrors.append(tr("A control has no name"));
```

Comment:
use addFormError in place of _formErrors.append: if not, no signal would be emitted.

Reply:
Oops!